server/protocols/jmap/src/main/java/org/apache/james/jmap/model/VacationResponse.java

```diff
@@ -71,7 +71,7 @@ public Builder textBody(String textBody) {
         }
 
         public Builder fromVacation(Vacation vacation) {
-            this.id = vacation.getId();
+            this.id = Vacation.ID;
```

Comment:
Fine, I saw you validate the id in the method, which is a good choice for correct error handling.